draft-ietf-httpbis-digest-headers.md

```diff
@@ -318,6 +327,79 @@ Want-Digest: sha-256
 Want-Digest: sha-512;q=0.3, sha-256;q=1, unixsum;q=0
 ~~~
 
+
+# The Content-Digest Field {#content-digest}
```

Comment:
So, strictly speaking, this field name is appropriate according to the current terminology in Semantics. 

What triggered my unease was that the `Content-` prefix is used on several headers that _don't_ refer to the actual content of the message (using the current Semantics definition) -- they refer to the _representation_. See <https://github.com/httpwg/http-core/issues/878>.

I think it's safe to ship this as `Content-Digest` as long as it's being super-precise about what it's referring to.

I could also see flipping `Content-Digest`'s semantics with `Digest`, to align with most (but not all) other uses of the `Content-` prefix. I'm not convinced that doing so is necessary, or even preferable; just mentioning it as something to consider.

Reply:
Good insight Mark, we'll provide an example of Content-Digest vs Digest in the same message.

About flipping, I think that Content-Digest, being similar to Content-md5 seems to me a more intuitive name.

Reply:
Flipping names seems bad to me. Prefer the way we suggested

Reply:
Wouldn't flipping the names be problematic with respect to compatiblity with existing implementations of RFC3230's Digest?